[Bug?]: react package failing with missing `helpers.mjs` file

### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Build is breaking due to bath styled-system JSR package.json

```bash
./node_modules/@cerberus-design/styled-system/css/conditions.mjs:1:0
Module not found: Can't resolve '../helpers.mjs'

<https://nextjs.org/docs/messages/module-not-found>

Import trace for requested module:
./node_modules/@cerberus-design/styled-system/css/css.mjs
./node_modules/@cerberus-design/styled-system/css/index.mjs
./src/pages/_app.js
```

### Expected behavior 🤔

_No response_

### Steps to reproduce 🕹

Steps:

1.
2.
3.
4.


### Context 🔦

_No response_

### Your environment 🌎

_No response_

--- a/.github/workflows/publish-next.yml
+++ b/.github/workflows/publish-next.yml
@@ -1,10 +1,9 @@
 name: 🚀 Publish Next Version
 
-on: [merge_group, pull_request]
-# on:
-#   push:
-#     branches:
-#       - main
+on:
+  push:
+    branches:
+      - main
 
 env:
   CI: true

--- a/.github/workflows/publish-next.yml
+++ b/.github/workflows/publish-next.yml
@@ -80,25 +80,25 @@ jobs:
   #     - run: pnpm dlx jsr publish --allow-dirty
   #       working-directory: ./packages/icons
 
-  publish-styled-system-jsr-next:
-    runs-on: ubuntu-latest
-    name: Publish Styled-System Next Version to JSR
-    permissions:
-      contents: read
-      id-token: write
-    steps:
-      - uses: actions/checkout@v4
-      - uses: oven-sh/setup-bun@v1
-      - uses: pnpm/action-setup@v4
-        with:
-          version: 9.1.4
-      - uses: actions/setup-node@v4
-        with:
-          cache: 'pnpm'
+  # publish-styled-system-jsr-next:
+  #   runs-on: ubuntu-latest
+  #   name: Publish Styled-System Next Version to JSR
+  #   permissions:
+  #     contents: read
+  #     id-token: write
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - uses: oven-sh/setup-bun@v1
+  #     - uses: pnpm/action-setup@v4
+  #       with:
+  #         version: 9.1.4
+  #     - uses: actions/setup-node@v4
+  #       with:
+  #         cache: 'pnpm'
 
-      - run: pnpm install
-      - run: pnpm run publish:jsr --next --commit ${{ github.sha }}
-        working-directory: ./configs
+  #     - run: pnpm install
+  #     - run: pnpm run publish:jsr --next --commit ${{ github.sha }}
+  #       working-directory: ./configs
 
-      - run: pnpm dlx jsr publish --allow-dirty --allow-slow-types
-        working-directory: ./packages/styled-system
+  #     - run: pnpm dlx jsr publish --allow-dirty --allow-slow-types
+  #       working-directory: ./packages/styled-system

--- a/.github/workflows/publish-next.yml
+++ b/.github/workflows/publish-next.yml
@@ -1,9 +1,10 @@
 name: 🚀 Publish Next Version
 
-on:
-  push:
-    branches:
-      - main
+on: [merge_group, pull_request]
+# on:
+#   push:
+#     branches:
+#       - main
 
 env:
   CI: true

--- a/.npmrc
+++ b/.npmrc
@@ -1,1 +1,0 @@
-@jsr:registry=https://npm.jsr.io

--- a/configs/src/versions.mjs
+++ b/configs/src/versions.mjs
@@ -1,4 +1,4 @@
-export const version = '0.1.1'
+export const version = '0.1.2'
 export const nextTag = 'next'
 
 export const packages = ['panda-preset', 'icons', 'react', 'styled-system']

--- a/docs/app/preset/overview.mdx
+++ b/docs/app/preset/overview.mdx
@@ -21,7 +21,8 @@ pnpm add -D @pandacss/preset-panda
 Install the Cerberus core packages, with the package manager of your choice:
 
 ```bash
-pnpm dlx jsr add @cerberus/{panda-preset,styled-system}
+pnpm dlx jsr add @cerberus/panda-preset
+pnpm -D add @cerberus/styled-system@npm:@cerberus-design/styled-system
 ```
 
 ## Step 2 - Setup Cerberus
@@ -39,7 +40,7 @@ export default defineConfig({
   ...cerberusConfig,
 
   include: [
-    './node_modules/@cerberus-design/react/src/**/*.{ts,tsx}',
+    './node_modules/@cerberus/react/src/**/*.{ts,tsx}',
     './app/**/*.{ts,tsx}', // <- Make sure this path is to your project
   ],
   exclude: [],
@@ -94,7 +95,7 @@ const poppins = Poppins({
 If you are interested in using the Cerberus React, install it now:
 
 ```bash
-pnpm add @cerberus-design/react
+pnpm add @cerberus/react@npm:@cerberus-design/react
 ```
 
 ### Using the Cerberus Icons (optional)
@@ -113,52 +114,20 @@ To help make maintaining a breeze, add this new script to your `package.json` to
 
 ```json title="package.json"
 "scripts": {
-  "up:cerberus": "pnpm up @cerberus-design/react@latest && pnpm dlx jsr add @cerberus/{panda-preset,styled-system,icons}"
+  "up:cerberus": "pnpm up @cerberus/{styled-system,react}@latest && pnpm dlx jsr add @cerberus/{panda-preset,icons}"
 }
 ```
 
-## Typescript
+## FAQ
 
-If you are seeing TypeScript path errors, you might need to take the following steps:
+### What are the weird version paths for when I install the Cerberus packages?
 
-### Step 1 - Update your tsconfig.json
+The Cerberus packages are published under the `@cerberus-design` organization in NPM (due to the name `cerberus` being taken). However, in JSR, we use the `@cerberus` organization. We eventually plan on fully migrating to JSR when there is better support for the features we need.
 
-```json title="tsconfig.json"
-{
-  "compilerOptions": {
-    "baseUrl": ".",
-    "paths": {
-      "@cerberus/styled-system/*": [
-        "./node_modules/@cerberus-design/styled-system/*"
-      ]
-    }
-  }
-}
-```
-This will map the JSR styled-system to the NPM expected location.
+The version paths are simply creating an alias to the `@cerberus-design` organization so that you can have consistent package naming across your project.
 
-### Step 2 - Add a `global.d.ts` file
+Everything should use the `@cerberus` organization in your code.
 
-```ts title="global.d.ts"
-declare module '@cerberus/styled-system/css/index.mjs' {
-  export * from '@cerberus-design/styled-system/css'
-}
+### Why do I need to add the `@cerberus/styled-system` package?
 
-declare module '@cerberus/styled-system/recipes/index.mjs' {
-  export * from '@cerberus-design/styled-system/recipes'
-}
-
-declare module '@cerberus/styled-system/types' {
-  export * from '@cerberus-design/styled-system/types'
-}
-
-declare module '@cerberus/styled-system/utils/index.mjs' {
-  export * from '@cerberus-design/styled-system/utils'
-}
-
-declare module '@cerberus/styled-system/patterns/index.mjs' {
-  export * from '@cerberus-design/styled-system/patterns'
-}
-
-```
-This will map the JSR styled-system to the NPM expected location.
+The `@cerberus/styled-system` package is a dependency of the Cerberus Panda Preset. It is a styled-system that is pre-configured to work with the Cerberus Design System and creates a single source of truth for our UI related packages.

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -14,7 +14,7 @@
   },
   "peerDependencies": {
     "@cerberus/panda-preset": "npm:@jsr/cerberus__panda-preset",
-    "@cerberus/styled-system": "npm:@cerberus-design/styled-system",
+    "@cerberus/styled-system": "npm:@cerberus-design/styled-system@next",
     "@pandacss/dev": "*",
     "react": "*",
     "react-dom": "*"

--- a/packages/styled-system/jsr.json
+++ b/packages/styled-system/jsr.json
@@ -2,11 +2,11 @@
   "name": "@cerberus/styled-system",
   "version": "0.0.0",
   "exports": {
-    "./css": "./css/index.d.ts",
-    "./patterns": "./patterns/index.d.ts",
-    "./recipes": "./recipes/index.d.ts",
-    "./jsx": "./jsx/index.d.ts",
-    "./tokens": "./tokens/index.d.ts"
+    "./css": "./css/index.mjs",
+    "./patterns": "./patterns/index.mjs",
+    "./recipes": "./recipes/index.mjs",
+    "./jsx": "./jsx/index.mjs",
+    "./tokens": "./tokens/index.mjs"
   },
   "publish": {
     "exclude": [

--- a/packages/styled-system/panda.config.ts
+++ b/packages/styled-system/panda.config.ts
@@ -8,5 +8,7 @@ export default defineConfig({
   exclude: [],
   outdir: '.',
 
+  clean: true,
+
   presets: [pandaPreset, cerberusPreset],
 })

